activities/management/commands/verification_date_to_be_adoption_date.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime 
+from django.db.models import Q, F
+from django.db import IntegrityError
+from django.core.management.base import BaseCommand
+from activities.models import *
+
+
+class Command(BaseCommand):
+	help = 'Saves date_of_verification in date_of_adoption field under PersonAdoptPractice'
+
+	def handle(self, *args, **options):
+		obj = PersonAdoptPractice.objects.filter(date_of_adoption__gte=datetime.datetime(2016, 07, 01)).exclude(date_of_verification__exact=None)\
```

Comment:
we can just have those objects where date of adoption is not equal to date of verification and date of verification is not null.